scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -392,11 +401,26 @@ private int getTotalNumberOfJobs(final DBJobDataParameters params) {
                     queryString.append("and parentId = null ");
                 }
 
+                if (!params.getJobName().isEmpty()) {
+                    queryString.append("and lower(jobName) like :jobName ");
```

Comment:
Lower will break the index. This means that the search will have to go through all DB lines, are you sure we want this ?

Reply:
I suggest also to not be case sensitive

Reply:
@mboussaa you mean to be case sensitive
Case insensitive search has very bad performances.

It is possible to implement it (by duplicating a column inside the database with lowercase data, but this requires additional work, notably regarding backward compatibility)